src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,143 @@
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Input from '../../common/Input/Input';
+import Button from '../../common/Button/Button';
+import Textarea from '../../common/Textarea/Textarea';
+import { mockedAuthorsList } from '../../constants';
+import pipeDuration from '../../helpers/pipeDuration';
+import './createCourse.css';
+
+function CreateCourse({ courseData, setCourseData, handleSubmit }) {
+	const [authorName, setAuthorName] = useState('');
+
+	const [authors, setAuthors] = useState(mockedAuthorsList);
+	const authorsList = authors.map((author) => {
+		return (
+			<div className='author' key={author.id}>
+				<p>{author.name}</p>
+				<Button
+					buttonText='Add author'
+					type='button'
+					handleClick={() => {
+						setCourseData({
+							...courseData,
+							authors: [
+								...courseData.authors,
+								{ id: author.id, name: author.name },
+							],
+						});
+						setAuthors((prev) => [
+							...prev.filter((item) => !(item.id === author.id)),
+						]);
```

Comment:
Use a render function for this purpose (I think I already provided an example for similar case somewhere earlier)